src/raster/r.in.usgs/r.in.usgs.py

```diff
@@ -880,7 +880,7 @@ def exist_list():
                             resolution_value=product_resolution,
                             extent="region",
                             resample=product_interpolation,
-                            memory=memory,
+                            memory=float(memory) / int(nprocs),
```

Comment:
Memory parameter of r.import should be integer.

Reply:
If I understand correctly, both `//` and additional `int()` is needed to get that right. `//` makes it clear I want to floor divide, `int()` takes the result (which is float) and removes the decimal part (which is `.0`).